modules/bitgo/test/v2/unit/internal/tssUtils/ecdsa.ts

```diff
@@ -301,35 +356,44 @@ describe('TSS Ecdsa Utils:', async function () {
       const backupShareHolder: BackupKeyShare = {
         userHeldKeyShare: backupKeyShare,
       };
-      const bitgoKeychain = await tssUtils.createBitgoKeychain(userGpgKey, userKeyShare, backupShareHolder);
+      const bitgoKeychain = await tssUtils.createBitgoKeychain({
+        userGpgKey,
+        userKeyShare,
+        backupKeyShare: backupShareHolder,
+        bitgoPublicGpgKey: bitgoPublicKey,
+      });
       bitgoKeychain.should.deepEqual(nockedBitGoKeychain);
       const testKeyShares = otherKeyShares;
       const testCasesPromises = [
```

Comment:
are the values here still promises?

Reply:
They're not lol. I remember adding an `await` because the IDE was having an eslint complaint. I'll remove them.